dependencyManagement/build.gradle.kts

```diff
@@ -33,7 +36,7 @@ val DEPENDENCY_BOMS = listOf(
   "com.fasterxml.jackson:jackson-bom:2.12.3",
   "com.google.guava:guava-bom:30.1.1-jre",
   "org.codehaus.groovy:groovy-bom:${groovyVersion}",
-  "io.opentelemetry:opentelemetry-bom:${otelVersion}",
+  "io.opentelemetry:opentelemetry-bom:${otelVersionSnapshot}",
```

Comment:
To avoid snapshot problems how about you copy the annotation to the new `instrumentation-annotation-support` module and add an issue for replacing it with the real one once SDK 1.4.0 is released? I'll add that issue to 1.4.0 milestone and make sure it's done before instrumentation repo releases.